torchinductor/scheduler.py

```diff
@@ -459,87 +537,237 @@ def create_fx_from_buffers(nodes, fname, print_graph=False):
     from torch.fx.passes.shape_prop import TensorMetadata
     from torch.fx.passes.tools_common import legalize_graph
 
-    func_dict = {}
-    name_to_fx_node = {}
-    graph = torch.fx.Graph()
-    first_node = None
+    graph = create_fx_from_snodes(nodes)
 
-    # create call_function node for each Buffer and Kernel
-    for snode in nodes:
-        node = snode.node
-        name = node.get_name()
-        node_type = str(type(node)).split(".")[-1].replace("'>", "")
-
-        if node_type in func_dict:
-            fake_f = func_dict[node_type]
-        else:
-            fake_f = get_fake_func(node_type)
-            func_dict[node_type] = fake_f
-        fx_node = graph.call_function(fake_f, args=(), kwargs=None)
-        fx_node.name = name
+    for node in graph.nodes:
+        if "fusion_meta" not in node.meta:
+            continue
+        group = node.meta["fusion_meta"].group
 
         # gather meta data
         dtype = None
         if isinstance(node, ir.ComputedBuffer):
             dtype = node.data.dtype
 
-        try:
-            stride = node.get_stride()
-        except AttributeError:
-            stride = None
+        metadata = TensorMetadata(group, dtype, None, None, None, None, None)
+        node.meta["tensor_meta"] = metadata
 
-        layout = type(node.layout)
+    if print_graph:
+        print(graph)
+    print("starting creating module")
```

Comment:
Remove prints before landing, or convert to logging calls.

Reply:
These only show up in debug code (i.e. pass `INDUCTOR_SCHEDULER_GRAPH=1`).